pkg/performanceprofile/cmd/render/render.go

```diff
@@ -162,7 +190,35 @@ func render(inputDir, outputDir string) error {
 			if err != nil {
 				return err
 			}
+			klog.Info(fileName)
+		}
+	}
+
+	return nil
+}
+
+func genCPUPartitioningManifests(pinnedMode *apicfgv1.CPUPartitioningMode, outputDir string, mcpNames ...string) error {
```

Comment:
This generates Workload partitioning manifests, right? I suggest a change of name here and maybe a short overall comment explaining the function and the use case.

Reply:
The name should probably make the configured component and generated object obvious. Is this method used for both crio and kubelet (253 and 260 in getIgnitionConfig?

Reply:
Updated with comments, this method is the bootstrap method during render so it's the one that generates the default state, the regular `GetNewComponents` takes on the responsibility of generating the configs derived from normal operations when customers supply a PerformanceProfile.